src/kiwixapp.cpp

```diff
@@ -438,6 +438,8 @@ void KiwixApp::createActions()
     CREATE_ACTION_SHORTCUT(ToggleTOCAction, gt("table-of-content"), QKeySequence(Qt::CTRL | Qt::SHIFT | Qt::Key_1));
     HIDE_ACTION(ToggleTOCAction);
 
+    CREATE_ACTION_ICON_SHORTCUT(OpenMultiZimAction, "filter", gt("search-options"), QKeySequence(Qt::CTRL | Qt::Key_M));
```

Comment:
@kelson42 Is the CTRL+M shortcut OK for a button named "Search options"?

Reply:
I was aware of this as well. I will look around to find an appropriate one or what is the usual shortcut for this.

Reply:
Done, switched to Ctrl + Shift + L which is a common shortcut for opening and closing search options.